action.yml

```diff
@@ -2,14 +2,45 @@ name: "Install Edgee's CLI"
 description: "Install edgee from www.github.com/edgee-cloud/edgee"
 author: "edgee-cloud"
 
+inputs:
+  build:
+    description: "Build Edgee from source instead of download"
+    required: false
+    default: "false"
```

Comment:
doesn't yaml support booleans too?

Reply:
looks like it isn't possible https://github.com/actions/runner/issues/1483

well, whatever 😆

Reply:
but what happens if someone uses this action with:

```
with:
    build: true
````
?